Mathlib/Data/Fin/Basic.lean

```diff
@@ -530,6 +532,12 @@ theorem nontrivial_iff_two_le : Nontrivial (Fin n) ↔ 2 ≤ n := by
 -- porting note: here and in the next lemma, had to use `← Nat.one_eq_succ_zero`.
 #align fin.nontrivial_iff_two_le Fin.nontrivial_iff_two_le
 
+theorem bot_ne_top : (⊥ : Fin (n + 2)) ≠ ⊤ := _root_.bot_ne_top
+
+theorem bot_ne_top_iff_two_le [NeZero n] : (⊥ : Fin n) ≠ ⊤ ↔ 2 ≤ n := by
```

Comment:
There ought to be a version elsewhere that says
`(⊥ : X) ≠ ⊤ ↔ NonTrivial X`

Reply:
We have `bot_ne_top`, but we don't seem to have an `iff` version?

Reply:
We have `subsingleton_iff_bot_eq_top` but I am not sure we have `nontrivial_iff_bot_ne_top`.

Reply:
I have removed this.